upgrading/upgrade-running-applications/2_new_feature.md

```diff
@@ -48,7 +48,7 @@ and edit the `../forum/daml.yaml`{{open}} file to update the `create-daml-app` d
 Your `daml.yaml` file should look like
 
 <pre class="file" data-target="clipboard">
-sdk-version: 1.1.1
+sdk-version: 1.2.0
```

Comment:
also would leave a TODO here.

Reply:
I got a reply about updating the HTML instructions via a variable today:

"Thanks for the suggestion. This is on the roadmap but it's not currently available. We'll keep you updated and let you know when it's available."